charts/tekton-apps/templates/eventlistener.yaml

```diff
@@ -17,32 +17,10 @@ spec:
             nodeSelector:
               {{- toYaml .Values.nodeSelector | nindent 14 }}
   {{- end }}
+  {{- if .Values.eventlistener.labelSelector }}
+  labelSelector:
+    matchLabels:
+      {{- .Values.eventlistener.labelSelector | toYaml | nindent 6 }}
+  {{- end }}
 
-  triggers:
-  {{- range $project := .Values.apps }}
-  {{- if $project.enabled }}
-  # ========================================================================
-  # {{ $project.project | upper }} PROJECT
-  {{- range $component := $project.components }}
-
-  # initialize `projectEnvironment` variable with either `$project.environment` if it exists or with `$.Values.environment` otherwise
-  {{ $projectEnvironment := ternary $project.environment $.Values.environment (hasKey $project "environment") -}}
-
-  # initialize `gitBranchPrefixes` variable with either `$component.eventlistener.gitWebhookBranches` list if it exists or `$.Values.gitBranchPrefixes` otherwise
-  {{ $gitBranchPrefixes := ternary $component.eventlistener.gitWebhookBranches $.Values.gitBranchPrefixes (hasKey $component.eventlistener "gitWebhookBranches") -}}
-
-  # ========================================================================
-  # 🐤 {{ $project.project | upper }}/{{ $component.name | upper }} COMPONENT
-  # ========================================================================
-
-  {{- $data := dict "project"           $project
-                    "component"         $component
-                    "environment"       $projectEnvironment
-                    "eventlistener"     $.Values.eventlistener
-                    "gitBranchPrefixes" $gitBranchPrefixes }}
-  {{- include "tekton-apps.eventlistener.trigger" $data | nindent 4 }}
```

Comment:
I suppose `tekton-apps.eventlistener.trigger` snippet can be removed now?

Reply:
Strictly saying, yes.